### PR TITLE
Implementation of dinamic data by API service

### PR DESCRIPTION
**Pull request by matheusmelchiades**
* Created method to get data from api 
* Created method to treat the data
* Fix the component "ForecastDetails" with static data to receive the dinamic data from API weather

*Obs: To merge this feature, was based in changes of #7 , so need merge that branch before.

### CODE REVIEW

**Comment by matheusmelchiades** [comment]
Hey @murilokaos, thanks by your comment. 
But when i did the feature, i saw that in PR #4 was included the google maps and to avoid merge i thought it would be better not include now hehehe :grin:.

File: src/services/api.js [hunk 1]
```diff
@@ -4,4 +4,42 @@ const api = axios.create({
   baseURL: 'https://api.github.com',
```
Comment (fernandovfilho) [inline src/services/api.js]: Change the address to use OpenWeather API https://openweathermap.org/current

Comment (matheusmelchiades) [reply src/services/api.js]: But you already changed this URL, #4 . I didn't change to not have conflits

Comment (fernandovfilho) [reply src/services/api.js]: This PR was closed

Comment (matheusmelchiades) [reply src/services/api.js]: Yeah, i saw that in the master already changed as doc the OpenWeather.
https://github.com/lucasmontano/openweathermap-reactnative/blob/ca3ee8d1385ab48f082454626e7cca06f64e60d9/src/services/api.js#L6

**Comment by AndersonAltissimo** [comment]
Hi @matheusmelchiades 

To increase the project organization we close all opened PRs and create a new way to contribute using issues and PR templates and a kanban github project. Your PR is now based on Issue #31 and Issue #32. Please follow the README instructions and open a new PR, we awaiting your contribution!

**Pull request closed without merging**